docs/faq.rst

```diff
@@ -0,0 +1,114 @@
+.. _faq:
+
+Frequently asked questions
+==========================
+
+General fairness questions
+--------------------------
+
+Where can I learn more about fairness in machine learning?
+    For a high level view of the subject, please see the :ref:`further reading <further_reading>`
+    section, where we provide links to a variety of books and papers.
+    In our :ref:`user guide <user_guide>` we also have links to the papers describing
+    Fairlearn's mitigation algorithms.
+
+Why not just ignore the sensitive features?
+    Because ignoring the sensitive feature usually doesn't work.
+
+    To elaborate, information is often redundantly encoded in datasets, and machine learning
+    algorithms *will* uncover these links (it is what they are designed to do).
+    For example, in the US, the ZIP code where a person lives is well correlated with their
+    race.
+    Even if the model is not provided with race as a feature, the model will pick up on it
+    implicitly via the ZIP code (and other features).
+    Worse, without having the race available in the dataset, it is not possible to determine
+    whether any use of ZIP code was as a proxy for race, or for some other reason.
+
+The model is unfair because the data are biased. Isn't it better to get better data?
+    If a model is unfair due to biased data (note that this is not the only reason a model
+    may be unfair), then obtaining unbiased data is definitely the best solution.
+    However, doing so might not always be practical.
+    In these cases, the mitigation techniques in Fairlearn and similar packages can redistribute
+    harms between subgroups in the data.
+    The extent to which this is 'fair' will depend on how the harms are redistributed, and the
+    broader context of the system of which the model is a part.
+
+Won't making a model fairer reduce its accuracy?
+    Any technique which reduces disparity in some metric is most likely to impact other metrics.
```

Comment:
Actually I would probably just use the answer that I gave [above](https://github.com/fairlearn/fairlearn/pull/462/files#r444376612) .

Reply:
I have done so.